tests/config/test_search.py

```diff
@@ -60,3 +60,96 @@ def test_search_config_init(mocker, locale, domain, exp_domain):
 
         # Assertions
         assert cfg.domain == exp_domain
+
+
+@pytest.mark.parametrize('keywords, province_or_state, locale, providers, in_city',
+                         [(['Ice Cream', 'Spiderman'], None, Locale.USA_ENGLISH, [enums.Provider.INDEED], 'Austin')])
+def test_search_config_validate_invalid_province(keywords, province_or_state, locale, providers, in_city):
+    cfg = SearchConfig(keywords, province_or_state, locale, providers, city=in_city)
+
+    with pytest.raises(AssertionError) as e:
```

Comment:
you can actually do this as `pytest.raises(AssertionError, match=r"Province/State not set")`

Reply:
Has been added on 0a9dd3102db59ce200b304a2e463301ab2e6d616